.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the .idea folder shouldn't be in the commit please update the `.gitignore` with this: 
https://github.com/Terasology/Rails/blob/master/.gitignore

you will also need to remove everything from the local cache and then re-add everything
```
git rm -rf --cached .
git add -A .
git commit "added gitignore"
```